### PR TITLE
Improved settings up to use Amazon S3 to static files and the CollectFast library.

### LINKED ISSUE #73: Setting Up the project to use S3 of AWS and the CollectFast library

### PR DESCRIPTION
**Pull request by Riverfount**
Closes #73

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,5 @@ ENV/
 
 #dynaconf settings
 /settings.yml
+5488499.png
```
Comment (renzon) [inline .gitignore]: I think you've added this entries while testing locally. Please remove since they do no apply for every contributor.

File: pythonpro/core/templates/core/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+<meta charset="UTF-8">
```
Comment (renzon) [inline pythonpro/core/templates/core/index.html]: This change is not related to the issue, can be removed as well

**Comment by renzon** [comment]
@Riverfount Also would be nice include AWS env var to env-sample ;)

**Comment by renzon** [comment]
I've noticed another issue: since you created another requirements file, it will not be loaded by travis. In fact I would those dependencies in requirement.txt itself, maybe you could only add a comment to separate them from the rest of libs.

**Comment by Riverfount** [comment]
@renzon I think that Travis-CI read the new requirements file because I put the line `-r requirements-aws.txt` in the file requiresments.txt. Well, that's works well in my WTTD project! ;-)
